app/test_split_integer.py

```diff
@@ -2,20 +2,36 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 15
+    number_of_parts = 3
+    result = split_integer(value, number_of_parts)
+    assert sum(result) == value
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    value = 20
+    number_of_parts = 5
```

Comment:
Do not create additional variables, pass values directly to the function and make required actions in the `assert`.